python/ray/serve/batching.py

```diff
@@ -112,6 +113,29 @@ def __init__(
             self._handle_batch_task = self._loop.create_task(
                 self._process_batches(handle_batch_func)
             )
+        self.check_max_batch_size_bounded()
+
+    def check_max_batch_size_bounded(self):
+        """Helper to check whether the max_batch_size is bounded.
+
+        Log a warning to configure `max_ongoing_requests` if it's bounded.
```

Comment:
```suggestion
    def warn_if_max_batch_size_exceeds_max_ongoing_requests(self):
        """Helper to check whether the max_batch_size is bounded.

        Log a warning to configure `max_ongoing_requests` if it's bounded.
```
unclear what "bounded" means without context.

Reply:
this should be a private method as well